src/room.js

```diff
@@ -133,7 +133,7 @@ async function exitScene() {
 function applyProfile(playerRig) {
   playerRig.setAttribute("player-info", {
     displayName: store.state.profile.display_name,
-    avatar: store.state.profile.avatar || "#bot-skinned-mesh"
+    avatar: '#' + (store.state.profile.avatar || "botdefault")
```

Comment:
I think we should include the "#" in the avatar, this means its a "bundled" avatar but I should also be able to just point it at a URL (even if we dont have a UI for that yet)

Reply:
I knew this was temporary. It's easy to change, so I'm going to leave it to later. I don't think I've precluded URL-based avatars in any significant way.